ANSWERS.md

```diff
@@ -0,0 +1,15 @@
+- [ ] What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+    PropTypes are used to ensure that the data being passed through to a component is of the correct types.  This is important because we do not want, for example, to pass a number into a data field that needs a string to function properly.
+
+- [ ] Describe a life-cycle event in React?
+
+    A life-cycle event is an event made to trigger either after or before the completion of a step of the React life-style, such as ComponentDidMount, or ComponentWillUnmount.
```

Comment:
Yes, also check this defintion by Tyler Thompson, he described it really well: 

"A life-cycle event is one of a number of phases that a component goes through as it is being mounted, updated, and eventually unmonunted from a page. It starts with the mounting phase in which a render method is called and the component is being built. Then it moves to the updating phase where setState can be used to update the data and force a re-render. Finally there is unmounting when the component is removed from the screen."